feature-utils/poly-analysis/utils/status.js

```diff
@@ -0,0 +1,23 @@
+export function createSuccessStatus() {
```

Comment:
Is there any alternative to these three functions that return different objects with different schemas?